apps/roboshield/payload.config.ts

```diff
@@ -122,4 +122,7 @@ export default buildConfig({
     }),
   ] as any[],
   telemetry: process?.env?.NODE_ENV !== "production",
+  typescript: {
+    declare: false,
+  },
```

Comment:
What are the benefits of this?

Reply:
By default its set to true, so when we generate payload types at the end of the generated `payload-types.ts` we will get a module and an exported interface. 
```
declare module 'payload' {
  export interface GeneratedTypes extends Config {}
}
```

However when I do that , I end up getting a name collision on my end. 

<img width="599" alt="Screenshot 2024-07-17 at 10 24 56" src="https://github.com/user-attachments/assets/6a50f1f2-b2bd-496f-9110-932aedcbc92b">

Reply:
Good catch... it's a collision between `GeneratedTypes` from payload and what?

Reply:
Hmm....the auto `GeneratedTypes` are colliding with whats in  `node_modules/payload/dist/index.d.ts`

Reply:
I'll need to do further investigation on this will merge it, to main as it is